sessionCheck.js

```diff
@@ -113,7 +113,9 @@
             window.addEventListener("message", this.eventListenerHandle);
 
             if (this.subject) {
-                sessionStorage.setItem("sessionCheckSubject", this.subject);
+                var subjectMap = sessionStorage.getItem("sessionCheckSubject") ? JSON.parse(sessionStorage.getItem("sessionCheckSubject")) : {};
```

Comment:
Rather than continuing with the same sessionStorage key as earlier versions, it might be better to introduce a new scheme so there is no chance of collision between old and new versions. For example, we could use "sessionCheckSubject-" + this.authId as the sessionStorage item identifier.

Reply:
fixed